### PR TITLE
#164237041 Displays cards on the dashboard

### PR DESCRIPTION
**Pull request by mikenthiwa**
#### What does this PR do?
 - Display cards view for automation.
 - Redesign automation table.

#### Description of Task to be completed?
- Display cards containing details of developers.
- Redesigning the dashboard table containing developer and partner information.

#### How should this be manually tested?
- Git clone this repository: `https://github.com/andela/bp-esa-frontend.git`
- Checkout to: `ft-add-cards-164237041`
- Run `yarn install`
- Add`REACT_APP_BACKEND_URL=http://localhost:6050`
- Run `yarn start`

#### What are the relevant pivotal tracker stories?
[164237041](https://www.pivotaltracker.com/story/show/164237041)

#### Screenshots (If applicable)
Screenshot 2019-03-20 at 10 12 49

### CODE REVIEW

File: src/components/fellows/index.jsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            <span className="date">1/30/2019 1:41 PM</span>
+          </div>
+          <div className="status-band">
+            <span>
+              {card.freckleAutomations.status.toUpperCase()}
+              <button type="button">RETRY</button>
+            </span>
+          </div>
+          <div className="status-container">
+            <div>
```
Comment (matthewacha) [inline src/components/fellows/index.jsx]: lines 43 to57 and 59 to 94 keep repeating, you could create this as metadata and import that metadata and map it to generate these elements

File: src/__tests__/components/ReportPage.test.jsx [hunk 2]
```diff
@@ -498,50 +502,55 @@ describe('<ReportPage />', () => {
         .at(0)
         .find('.fellow');
       global.open = jest.fn();
+      console.log('>>>>>>>>>>', redirectToAIS);
```
Comment (matthewacha) [inline src/__tests__/components/ReportPage.test.jsx]: remove this

File: src/__tests__/components/ReportPage.test.jsx [hunk 3]
```diff
@@ -498,50 +502,55 @@ describe('<ReportPage />', () => {
...
+    //     .find('.fa.fa-info-circle.success')
+    //     .at(2)
+    //     .simulate('click');
+    //   expect(component.state('type')).toEqual('freckle');
+    // });
+
+    // it('should close modal', () => {
+    //   const component = mount(
+    //     <Provider store={store}>
+    //       <ReportPage {...props} />
```
Comment (matthewacha) [inline src/__tests__/components/ReportPage.test.jsx]: remove commented out code

File: src/components/ReportPage/index.jsx [hunk 4]
```diff
@@ -273,16 +264,16 @@ class ReportPage extends PureComponent {
...
           {this.statusBreakdown(report, type)}
-        </span>
-        <i
-          className={`fa fa-info-circle ${automationStatus}`}
-          onClick={() => { this.openModal(); this.changeModalTypes(report, type); }}
-        />
+        </div>
+        {/* <i */}
+        {/* className={`fa fa-info-circle ${automationStatus}`} */}
+        {/* onClick={() => { this.openModal(); this.changeModalTypes(report, type); }} */}
```
Comment (matthewacha) [inline src/components/ReportPage/index.jsx]: remove commented out code

File: src/components/ReportPage/index.jsx [hunk 5]
```diff
@@ -303,36 +294,41 @@ class ReportPage extends PureComponent {
...
           className="fellow"
           onClick={() => window.open(`https://ais.andela.com/people/${report.fellowId}`)}
-          title={report.fellowName}
-        >
+          title={report.fellowName}>
           {report.fellowName}
         </td>
         <td title={report.partnerName}>{report.partnerName}</td>
         <td>{report.type}</td>
         <td>{this.renderAutomationStatus(report.slackAutomations.status, report, 'slack')}</td>
```
Comment (matthewacha) [inline src/components/ReportPage/index.jsx]: please destructure report

File: src/components/ReportPage/styles.scss [hunk 6]
```diff
@@ -66,7 +98,7 @@
     background-color: rgb(248, 249, 251);
 
     p {
-      font-family: 'Din Pro';
+      font-family: 'Arial';
```
Comment (matthewacha) [inline src/components/ReportPage/styles.scss]: let's keep this at Din Pro

File: src/components/fellows/index.jsx [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+      const firstInitials = newName[0].charAt(0);
+      const secondInitials = newName[1].charAt(1);
+      return (
+        <div className="card" key={card.id}>
+          <div className="info-cont">
+            <img className="info-icon onBoarding-icon" src={onBoarding} alt="onboarding icon" />
+            <img className="info-icon" src={InfoIcon} alt="info icon" />
+          </div>
+          <div className="developerPicture">
+            <div className="oval-1">
```
Comment (matthewacha) [inline src/components/fellows/index.jsx]: adjust the position of the initials to be centred

**Comment by matthewacha** [comment]
Fix the codeclimate issue, we get this merged

**Comment by valmayaki** [comment]
@mikenthiwa @matthewacha I have refactored the pr to reduced the codeclimate issue

**Pull request merged**